q2_types/per_sample_sequences/_transformer.py

```diff
@@ -94,6 +95,17 @@ def _11(dirfmt: CasavaOneEightLanelessPerSampleDirFmt) \
         dirfmt, SingleLanePerSamplePairedEndFastqDirFmt, parse_lane=False)
 
 
+@plugin.register_transformer
+def _12(dirfmt: SingleLanePerSampleSingleEndFastqDirFmt) \
+        -> CasavaOneEightSingleLanePerSampleDirFmt:
+    casava = CasavaOneEightSingleLanePerSampleDirFmt(mode='w')
+    for file in dirfmt.path.iterdir():
+        if 'MANIFEST' not in str(file) and 'metadata' not in str(file):
```

Comment:
DBC: `file` is a `pathlib` object --- you can use the [`.match`](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.match) method to (among others) to check if the filename matches a given pattern.

Reply:
You could also just [`glob`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob) over all of the fastq.gz files, instead.